pgc_ndvi.py

```diff
@@ -52,6 +53,8 @@ def main():
                         help="submission script to use in PBS/SLURM submission (PBS default is qsub_ndvi.sh, SLURM "
                              "default is slurm_ndvi.py, in script root folder)")
     parser.add_argument("-l", help="PBS resources requested (mimicks qsub syntax, PBS only)")
+    parser.add_argument("--log", nargs='?', const="default",
+                        help="path to file to log progress (default is ortho_<timestamp>.log next to the <dst dir>")
```

Comment:
I think you need to clarify this help string. The way it reads now, it seem that leaving off `--log` will result in a log being written next to the `dst_dir`.  But I think you mean that if a person types `--log default` they will get that default log location. Is that correct?

Reply:
Yeah, good point. By default it will NOT write the log, but if you just give it `--log` without a specific path it writes it to that default location. I'll push a clarification for that.